lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -342,7 +344,12 @@ sub _stage_cromwell_output {
 
         my $destination = File::Spec->join($results_dir, $file);
         if (-e $destination) {
-            $self->fatal_message('Cannot stage results. Multiple outputs with identical names: %s', $file);
+            if( File::Compare::compare($source, $destination) == 0 ) {
```

Comment:
It looks like this does a line-by-line comparison - how big are the files in question and do we already have a hash or something we could use instead? This seems like it could be quite slow depending on the file size.

Reply:
Only `File::Compare::compare_text` goes line-by-line.  Plain old `compare` uses lower-level I/O to go through chunks of the file at a time.  It is still potentially slow (far slower than comparing existing hashes would be!), but I don't think we're currently hashing every output file.  That itself would be slower in practice, since we don't expect name collisions.  (And, if we do have name collisions, we normally expect the files to differ.)

In the one case we've encountered matching files like this, they were less than 1 KB.  (Only headers with no content.)